build/source/driver/multi_driver.f90

```diff
@@ -371,6 +389,25 @@ program multi_driver
 call mDecisions(err,message); call handle_err(err,message)
 
 ! *****************************************************************************
+! (3c) allocate space for output statistics data structures
+! *****************************************************************************
+! loop through data structures
+do iStruct=1,size(structInfo)
+ ! allocate space
+ select case(trim(structInfo(iStruct)%structName))
+  case('forc'); call allocStat(forc_meta , forcStat , err, message)   ! model forcing data
+  case('prog'); call allocStat(prog_meta , progStat , err, message)   ! model prognostic (state) variables
+  case('diag'); call allocStat(diag_meta , diagStat , err, message)   ! model diagnostic variables
+  case('flux'); call allocStat(flux_meta , fluxStat , err, message)   ! model fluxes
+  case('indx'); call allocStat(flux_meta , indxStat , err, message)   ! index vars
+  case('bvar'); call allocStat(bvar_meta , bvarStat , err, message)   ! basin-average variables
+  case default; cycle;
```

Comment:
Make spacing around ',' consistent with how it's done elsewhere in the same file.